src/modules/review/dto/create.review.dto.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  IsNotEmpty,
+  IsNumber,
+  IsString,
+  IsUUID,
+  Max,
+  Min,
+} from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class CreateReviewDto {
+  @ApiProperty({
+    description: 'Content of the review.',
+    example: 'Great accommodation, would stay again!',
+  })
+  @IsNotEmpty({ message: 'Content is required.' })
+  @IsString()
+  content: string;
+
+  @ApiProperty({
+    description: 'Rating for the accommodation (from 1 to 5).',
+    example: 4,
+    minimum: 1,
+    maximum: 5,
+  })
+  @Min(1, { message: 'Rating must be at least 1.' })
+  @Max(5, { message: 'Rating must be at most 5.' })
+  @IsNotEmpty({ message: 'Rating is required.' })
+  @IsNumber()
```

Comment:
It would make more sense if this decorator came before ```@Min()``` and ```@Max()```.
```@IsNotEmpty()``` => ```@IsNumber()``` => ```@Min()``` => ```@Max()```